src/lib/pages/nexus-endpoint.svelte

```diff
@@ -38,6 +42,7 @@
       </h1>
       <Button
         href={routeForNexusEndpointEdit(endpoint.id)}
+        hidden={editHidden}
```

Comment:
I don't believe this will hide the button. We don't have a hidden prop. I think it's fine to just use editDisabled and not hide this at all